rc/tools/ctags.kak

```diff
@@ -109,6 +109,8 @@ declare-option -docstring "shell command to run" \
     str ctagscmd "ctags -R --fields=+S"
 declare-option -docstring "path to the directory in which the tags file will be generated" str ctagspaths "."
 
+hook global EnterDirectory .* %{ nop %sh{ [ -d .tags.kaklock ] && rmdir .tags.kaklock } }
+
```

Comment:
probably it gets stuck in

    while ! mkdir .tags.kaklock 2>/dev/null; do sleep 1; done

how about `mkdir -p .tags.kaklock || exit` ? Maybe also check git blame info for this line